web/packages/core/src/public/v1.ts

```diff
@@ -0,0 +1,173 @@
+     *
+     * If this player has never been loaded, this method will return an error.
+     */
+    reload(): Promise<void>;
+
+    /**
+     * Loads a specified movie into this player.
+     *
+     * This will replace any existing movie that may be playing.
+     *
+     * @param options One of the following:
+     * - A URL, passed as a string, which will load a URL with default options.
+     * - A [[URLLoadOptions]] object, to load a URL with options.
+     * - A [[DataLoadOptions]] object, to load data with options.
+     * The options, if provided, must only contain values provided for this specific movie.
+     * They must not contain any default values, since those would overwrite other configuration
+     * settings with a lower priority (e.g. the general RufflePlayer config).
+     *
+     * The options will be defaulted by the [[config]] field, which itself
+     * is defaulted by a global `window.RufflePlayer.config`.
+     */
+    load(options: string | URLLoadOptions | DataLoadOptions): Promise<void>;
+
+    /**
+     * Plays or resumes the movie.
+     */
+    play(): void;
+
+    /**
+     * Whether this player is currently playing.
+     *
+     * @returns True if this player is playing, false if it's paused or hasn't started yet.
+     */
+    get isPlaying(): boolean;
+
+    /**
+     * Returns the master volume of the player.
+     *
+     * The volume is linear and not adapted for logarithmic hearing.
+     *
+     * @returns The volume. 1.0 is 100% volume.
+     */
+    get volume(): number;
+
+    /**
+     * Sets the master volume of the player.
+     *
+     * The volume should be linear and not adapted for logarithmic hearing.
+     *
+     * @param value The volume. 1.0 is 100% volume.
+     */
+    set volume(value: number);
+
+    /**
+     * Checks if this player is allowed to be fullscreen by the browser.
+     *
+     * @returns True if you may call [[enterFullscreen]].
+     */
+    get fullscreenEnabled(): boolean;
+
+    /**
+     * Checks if this player is currently fullscreen inside the browser.
+     *
+     * @returns True if it is fullscreen.
+     */
+    get isFullscreen(): boolean;
+
+    /**
+     * Exported function that requests the browser to change the fullscreen state if
+     * it is allowed.
+     *
+     * @param isFull Whether to set to fullscreen or return to normal.
+     */
+    setFullscreen(isFull: boolean): void;
+
+    /**
+     * Requests the browser to make this player fullscreen.
+     *
+     * This is not guaranteed to succeed, please check [[fullscreenEnabled]] first.
+     */
+    enterFullscreen(): void;
+
+    /**
+     * Requests the browser to no longer make this player fullscreen.
+     */
+    exitFullscreen(): void;
+
+    /**
+     * Pauses this player.
+     *
+     * No more frames, scripts or sounds will be executed.
+     * This movie will be considered inactive and will not wake up until resumed.
+     */
+    pause(): void;
```

Comment:
This is actually Ruffle's suspend/resume, not Flash's play/pause. I suggest we rename this to `suspend` or `suspended = true` or something